### PR TITLE
Tiny improvements.

### PR DESCRIPTION
**Pull request by irfanevrens**

### CODE REVIEW

File: src/LinkedIn.php [hunk 1]
```diff
@@ -659,7 +659,8 @@ protected function filterRequestOption(array &$options)
         switch ($options['format']) {
             case 'simple_xml':
                 $options['simple_xml'] = true;
-            // simple_xml is still xml. This should fall through
+                $options['headers']['Content-Type'] = 'text/xml';
+                break;
```
Comment (Nyholm) [inline src/LinkedIn.php]: Why do you want to duplicate a line and add the `break`?

File: src/LinkedIn.php [hunk 2]
```diff
@@ -277,7 +277,7 @@ protected function getUserFromAccessToken()
         try {
             return $this->api('GET', '/v1/people/~:(id,firstName,lastName,headline)');
         } catch (LinkedInApiException $e) {
-            return;
+            return null;
```
Comment (Nyholm) [inline src/LinkedIn.php]: `return null` is redundant, you should use ´return´ only. This is recommended by sensiolabs insight if I remember correctly.

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 phpunit.xml
 composer.lock
 vendor
+.idea
```
Comment (Nyholm) [inline .gitignore]: It is considered best prectise that IDE specific 'ignores' should be in you systems global ignore file and not in a specific project.

**Comment by Nyholm** [comment]
Thank you for sending a PR. I really like the changes to the LoginException. Some of your other changes is conflicting with best practices.

File: src/LinkedIn.php (hunk 2 above)
Comment (irfanevrens) [inline src/LinkedIn.php]: To show what you're returning is a best practice. If you are not agree with me, I can erase them.

File: src/LinkedIn.php (hunk 1 above)
Comment (irfanevrens) [inline src/LinkedIn.php]: If a case is not empty, we should use break for a logical rule on "switch case"

File: src/LinkedIn.php (hunk 2 above)
Comment (Nyholm) [inline src/LinkedIn.php]: It is a part of the [PSR fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) and the Symfony coding standards. 

Please erase them (or run PSR fixer and let it fix it for you)

File: src/LinkedIn.php (hunk 1 above)
Comment (Nyholm) [inline src/LinkedIn.php]: Somehow I really like not using the break here. But you are right, it is more clear and it gets easier to read.

**Comment by Nyholm** [comment]
Thank you!

**Pull request merged**